lib/release_notes_bot_web/controllers/slack_interaction_controller.ex

```diff
@@ -13,9 +13,9 @@ defmodule ReleaseNotesBotWeb.SlackInteractionController do
   end
 
   def index(conn, params) do
-    %{"trigger_id" => trigger, "view" => view, "user" => user} = Projects.parse_params(params)
```

Comment:
🤔

Reply:
Yeah I destructured this as a part of debugging. I can undo this since I know destructuring like this is idiomatic.

Reply:
Do you mean personal debugging or application-level debugging? I'd say let's drop this either way since you're right, destructuring is the preferred approach

Reply:
roger that - shes getting dropped.